mongorover/MongoCollection.lua

```diff
@@ -270,15 +270,15 @@ local MongoCollection = {__mode="k"}
 	-- Example usage at @{indexing.lua}.
 	-- @tparam table of fields and their corresponding index types
 	-- @tparam table of options to be used when creating index
-	-- @return string index name on success
+	-- @return string index name
 	function MongoCollection:createIndex(keys,opt)
 	    return self.collection_t:collection_create_index(luaBSONObjects, keys, opt)
 	end
 
 	---
 	-- Delete an index for a specified collection.
 	-- Example usage at @{indexing.lua}.
-	-- @tparam string an index name to be deleted
+	-- @tparam string an index name or original table with indexed fields and their corresponding types
```

Comment:
You are correct, and I knew that PyMongo was not returning anything. I think with this PR I was implementing your requested change to return Mongo Cursor for findIndexes().

However I think I also made it a bit confusing by also changing the comment line 281 that describes the input parameter into `dropIndex()` to properly describe the fact that input parameter into the `dropIndex()` function can be an index name or a keys table that was used to create an index in the first place. The input parameter is allowed to be a table in PyMongo. As far as the return value, We are still returning a boolean that we get from `mongoc_collection_drop_index()`. If you think this needs to change, please let me know.